holoviews/util/transform.py

```diff
@@ -47,6 +47,10 @@ def norm(values, min=None, max=None):
     """
     min = np.min(values) if min is None else min
     max = np.max(values) if max is None else max
+    if min==max:
+        if max==0:
+            return 0*values
+        return values/max
```

Comment:
If the values are already zero you don't need to multiply them by zero. Otherwise this looks good. Thank you!

```suggestion
    if min == max:
        return values if max == 0 else (values / max)
```